resources/packs/gcp/gcp.lr

```diff
@@ -172,6 +172,44 @@ private gcp.project.computeService {
   zones() []gcp.project.computeService.zone
   // List of backend services
   backendServices() []gcp.project.computeService.backendService
+  // List of IP addresses
+  addresses() []gcp.project.computeService.address
+}
+
+// GCP Compute Address
+private gcp.project.computeService.address @defaults("name address addressType") {
+  // Unique identifier
+  id string
+  // Static IP address
+  address string
+  // Address type
+  addressType string
+  // Creation timestamp
+  created time
+  // Address description
+  description string
+  // IP version
+  ipVersion string
+  // Endpoint type
+  ipv6EndpointType string
+  // Address name
+  name string
+  // URL of the network in which to reserve the address
+  networkUrl string
+  // Network tier used for configuring this address
+  networkTier string
+  // Prefix length if the resource represents an IP range
+  prefixLength int
+  // Address purpose
+  purpose string
+  // Region URL
+  regionUrl string
+  // Address status
+  status string
+  // URL of the subnetwork in which to reserve the address
+  subnetworkUrl string
+  // URLs of the resources that are using this address
+  userUrls []string
```

Comment:
What is the type here. `userUrl` is confusing to me. Is it returning different resource types or just a list of one type?

Reply:
Then it should be called `resourcesUrls` or probably better `resourceIds`

Reply:
That will deviate from the GCP API. They called it `users`. I try to have the properties closely related to the original API because then people can cross-reference them with the GCP docs if needed. I can rename that but not sure if it's a good idea